test_utils/src/lib.rs

```diff
@@ -14,11 +14,11 @@ use futures::Stream;
 use indexmap::IndexMap;
 use multiaddr::Multiaddr;
 use rand::{rngs::StdRng, Rng, SeedableRng as _};
+use std::sync::Arc;
 use std::{
     collections::{BTreeSet, VecDeque},
     ops::RangeInclusive,
     pin::Pin,
-    sync::Arc,
```

Comment:
that's strange, have you run `cargo fmt`?

Reply:
I have a file watcher that does this automatically